docs/current/sdk/go/snippets/multistage-build/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"dagger.io/dagger"
+)
+
+func main() {
+	// Create dagger client
+	ctx := context.Background()
+	client, err := dagger.Connect(ctx, dagger.WithLogOutput(os.Stdout))
+	if err != nil {
+		panic(err)
+	}
+	defer client.Close()
+
+	project := client.Host().Workdir()
+
+	// Build our app
+	builder := client.Container().
+		From("golang:latest").
+		WithMountedDirectory("/src", project).
```

Comment:
`WithMountedDirectory` takes a `dagger.DirectoryID` argument.

```suggestion
		WithMountedDirectory("/src", projectID).
```